src/js/components/Marathon.jsx

```diff
@@ -44,21 +46,24 @@ var Marathon = React.createClass({
   },
 
   componentDidMount: function () {
-    this.onRouteChange();
+    PluginActions.requestPlugins();
 
+    this.onRouteChange();
     this.bindKeyboardShortcuts();
 
-    PluginStore.once(PluginEvents.BOOTSTRAP_COMPLETE, () => {
-      this.startPolling();
-    });
-
-    PluginStore.bootstrap();
+    PluginStore.on(PluginEvents.CHANGE, this.onPluginStoreChange);
   },
 
   componentWillReceiveProps: function () {
     this.onRouteChange();
   },
 
+  onPluginStoreChange: function () {
+    if (PluginStore.getPluginLoadingState() > States.STATE_LOADING) {
```

Comment:
Just listen for the startup complete event of the other plugin. But, as discussed, I wouldn't recommend that, as the plugin manifest does not include dependencies nor any plugin order thus it's a very fragile.

```
  let pluginId = "another-cool-plugin"
  let dispatchToken = PluginDispatcher.register(function (event) {
          if (event.eventType === PluginEvents.STARTUP_COMPLETE
              && event.pluginId === pluginId) {
            PluginDispatcher.unregister(dispatchToken);
			 // Do something cool with the other plugin
          }
        });
```